src/core/symbology/qgsgraduatedsymbolrenderer.h

```diff
@@ -222,14 +222,66 @@ class CORE_EXPORT QgsGraduatedSymbolRenderer : public QgsFeatureRenderer
     Mode mode() const { return mMode; }
     void setMode( Mode mode ) { mMode = mode; }
 
+    /**
+     * get mUseSymmetricMode, which says if we want to classify symmetric around a given value
```

Comment:
* Don't reference to member variable names in docstrings, they are an implementation detail.
* Start docstrings with capital letters.
* Update to 3.4 (sorry)